app/src/main/java/com/io/movio/HiltApplication.kt

```diff
@@ -0,0 +1,7 @@
+package com.io.movio
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class HiltApplication:Application()
```

Comment:
the application is not Hilt its' Movio. This should be MovioApplication:Application()